src/new-client/src/app/site/[site]/site-page.tsx

```diff
@@ -22,14 +18,16 @@ export const SitePage = ({ site, permissions }: SitePageProps) => {
         </div>
       </div>
       <ul className="nhsuk-grid-row nhsuk-card-group">
```

Comment:
There used to be a check here to avoid rendering the `<ul>` element if there aren't any `<li>`s to show within it. This would occur if the user had none of the "permissionsRelevantToCards" for desperate want of a better term :D 
Could we add this back in? Otherwise we'll probably get browsers errors about an empty `<ul>`

Reply:
Yeah I can add that in, it wasn't there when I made these changes mind you. They were only on the `<li>` elements so the `<ul>` would have been rendered regardless.

Reply:
Yeah, I removed it when this card was added without permissions because at that time we could guarantee there'd always be at least 1 element in the list. I probably should've replaced it with a TODO tbf rather than just removing it temporarily